### ISSUE TITLE
kola: F36: aarch64: tests erroring occasionally with `ldconfig.service` failure

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#1169]
We've seen it twice now where `ldconfig.service` fails to start successfully on boot with a `status=11/SEGV` and a coredump. So far we've only seen it on `aarch64`. 

- [kola-openstack#183](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/job/kola-openstack/183/) `next-devel`/`36.20220330.10.0`
    - [kola-openstack-183-ext.config.firewall.iptables-legacy.journal.txt](https://github.com/coreos/fedora-coreos-tracker/files/8490846/kola-openstack-183-ext.config.firewall.iptables-legacy.journal.txt)
- [kola-aws#225](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/job/kola-aws/225/) `next-devel`/`36.20220412.10.1`
    - [kola-aws-225-coreos.ignition.once.journal.txt](https://github.com/coreos/fedora-coreos-tracker/files/8490847/kola-aws-225-coreos.ignition.once.journal.txt)

Both of these versions have `glibc-2.35-4.fc36`.

Some of the journal output around the time:

```
Apr 14 03:50:40.435989 systemd-coredump[1284]: Process 1277 (ldconfig) of user 0 dumped core.
                                                                                                                                                                                                                                                                     
                                               Module /usr/sbin/ldconfig with build-id 2a9b8a406f3ec08ac16783aab774d24d7accd4c9
                                               Module linux-vdso.so.1 with build-id 1f10a316ab1b4e2a184409f5063a6e7577f00b30
                                               Stack trace of thread 1277:
                                               #0  0x0000aaaab2179a4c brk (/usr/sbin/ldconfig + 0x39a4c)
                                               #1  0x0000aaaab2179adc sbrk (/usr/sbin/ldconfig + 0x39adc)
                                               #2  0x0000aaaab21506ac __libc_setup_tls (/usr/sbin/ldconfig + 0x106ac)
                                               #3  0x0000aaaab2150304 __libc_start_main_impl (/usr/sbin/ldconfig + 0x10304)
                                               #4  0x0000aaaab2149cb0 _start (/usr/sbin/ldconfig + 0x9cb0)
                                               ELF object binary architecture: AARCH64
                                                                                     
Apr 14 03:50:40.436400 systemd[1]: ldconfig.service: Main process exited, code=dumped, status=11/SEGV
```

**Comment by fweimer-rh** [comment]
We have seen this as well. I suspect it's a kernel bug in ASLR for static PIE binaries. Our Python maintainer attempted to raise it here:

* [brk() regression on AArch64 on static-pie binary -- issue with ASLR and a guard page? ](https://bugzilla.kernel.org/show_bug.cgi?id=215720)

**Comment by dustymabe** [comment]
According to [the BZ]() this is fixed in `kernel-5.17.4*`. In that case this should be fixed in all but the `stable` stream.
